code/__HELPERS/roundend.dm

```diff
@@ -281,6 +279,8 @@
 					L.actions = list()
 					L.mind?.spell_list = list()
 					L.a_intent_change(INTENT_HELP)
+				do_teleport(teleatom = L, destination = ERG_turf, channel = TELEPORT_CHANNEL_FREE)
+				SEND_SOUND(L, sound('modular_skyrat/sound/ambience/e1m1.mid', TRUE, 0, CHANNEL_ADMIN, 100))
```

Comment:
It makes no sense to play e1m1 for someone who isn't doing ERG, i assume that you're teleporting those who don't want to ERG.

Reply:
You're right! Corrected.